app/src/main/java/org/simple/clinic/search/PatientSearchScreen.kt

```diff
@@ -47,7 +48,7 @@ class PatientSearchScreen(context: Context, attrs: AttributeSet) : RelativeLayou
     }
     TheActivity.component.inject(this)
 
-    backButton.setOnClickListener {
+    toolbar.backButton.setOnClickListener {
```

Comment:
We need to in this case because the back button is the child of the toolbar, right? I am not sure if the kotlin synthetic view accessors will let you access it without the parent container.

Reply:
Looks like we don't. I'll remove this.